libr/core/cbin.c

```diff
@@ -1092,11 +1092,14 @@ static int bin_entry(RCore *r, int mode, ut64 laddr, int va, bool inifin) {
 		r_cons_printf ("[");
 	} else if (IS_MODE_NORMAL (mode)) {
 		if (inifin) {
-			r_cons_printf ("[Constructors]\n");
+			r_cons_printf ("[Constructors] ");
 		} else {
-			r_cons_printf ("[Entrypoints]\n");
+			r_cons_printf ("[Entrypoints] ");
```

Comment:
that extra space should go in the baddr line below not here

Reply:
emm ok 789ccb60af28fb60f0b4e397c0db7a307d773e62